fplus-database/src/database/applications.rs

```diff
@@ -44,45 +44,41 @@ pub async fn get_applications() -> Result<Vec<ApplicationModel>, sea_orm::DbErr>
     .all(&conn)
     .await?;
 
-    //Iterate over the results and convert them to ApplicationModel
-    let mut applications: Vec<ApplicationModel> = Vec::new();
-    for app in app_data {
-        applications.push(ApplicationModel {
-            id: app.get("id").unwrap().as_str().unwrap().to_string(),
-            owner: app.get("owner").unwrap().as_str().unwrap().to_string(),
-            repo: app.get("repo").unwrap().as_str().unwrap().to_string(),
-            pr_number: app.get("pr_number").unwrap().as_i64().unwrap(),
-            issue_number: app
-                .get("issue_number")
-                .expect("Issue number should be in the application data")
-                .as_i64()
-                .expect("Issue number must exist"),
-            application: Some(
-                app.get("application")
-                    .unwrap()
-                    .as_str()
-                    .unwrap()
-                    .to_string(),
-            ),
-            updated_at: Utc.from_utc_datetime(
-                &app.get("updated_at")
-                    .unwrap()
-                    .as_str()
-                    .unwrap()
-                    .parse::<DateTime<Utc>>()
-                    .unwrap()
-                    .naive_utc(),
-            ),
-            sha: Some(app.get("sha").unwrap().as_str().unwrap().to_string()),
-            path: Some(app.get("path").unwrap().as_str().unwrap().to_string()),
-            client_contract_address: app
-                .get("client_contract_address")
-                .map(|client_contract_address| client_contract_address.to_string()),
-        });
-    }
+    let applications = app_data
+        .into_iter()
+        .map(|app| ApplicationModel {
```

Comment:
Long-term (not this PR) it would be better to replace this with https://docs.rs/serde_json/latest/serde_json/fn.from_value.html